tests/robust/robust_fixtures.py

```diff
@@ -0,0 +1,208 @@
+import math
+from typing import Callable, Optional, TypeVar
+
+import pyro
+import pyro.distributions as dist
+import torch
+from pyro.nn import PyroModule
+
+from chirho.observational.handlers import condition
+from chirho.robust.internals.utils import ParamDict
+from chirho.robust.ops import Point
+
+pyro.settings.set(module_local_params=True)
+T = TypeVar("T")
+
+
+class SimpleModel(PyroModule):
```

Comment:
An easy way to add a bit more test coverage would be to parameterize `SimpleModel` by a link function that returns a `Distribution` for `y` and add more test cases with different link functions.